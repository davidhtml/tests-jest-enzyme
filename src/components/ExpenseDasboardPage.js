import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilter';
import ExpenseSummary from './ExpenseSummary';

const ExpenseDasboardPage = () => (
    <div>
        <ExpenseSummary />
        <ExpenseListFilters />
        <ExpenseList />
    </div>
);

export default ExpenseDasboardPage;
