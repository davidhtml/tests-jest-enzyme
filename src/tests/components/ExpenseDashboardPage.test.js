import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDasboardPage from '../../components/ExpenseDasboardPage';

test('should renturn ExpenseDasboardPage', () => {
    const wrapper = shallow(<ExpenseDasboardPage />);
    expect(wrapper).toMatchSnapshot();
});
