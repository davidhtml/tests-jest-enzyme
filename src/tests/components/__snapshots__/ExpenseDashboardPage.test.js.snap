// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renturn ExpenseDasboardPage 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
